aries-site/src/examples/components/tooltip/ToolTipSimpleExample.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box, Button, Tip, Text } from 'grommet';
+
+export const ToolTipSimpleExample = () => {
+  return (
+    <Box align="center" justify="center" fill>
```

Comment:
Is this outer Box wrapper necessary here? I think the Example component takes care of centering and we can just go straight to <Tip />

Reply:
You're right all this is taken care of with `Example` component